docs/terminal/shell-integration.md

```diff
@@ -81,6 +81,16 @@ code --locate-shell-integration-path bash
 [[ "$TERM_PROGRAM" == "vscode" ]] && . "/path/to/shell/integration/script.sh"
 ```
 
+## Shell integration quality
+
+When using shell integration, it has a "quality" associated with it that declares the capabilities of it. These qualities are determined by how the shell integration script behaves.
+
+- **None**: No shell integration is active
+- **Rich**: Shell integration is active and command detection is working in an ideal way.
```

Comment:
The use of "ideal" is vague - is there a way to make it more explicit what is supported?

Reply:
It's kind of elaborated below. Explaining what ideal means here is too technical imo as this section of the page is targeting a user of shell integration, not a shell developer